src/userobject/ElasticRecoilCrossSectionUserObject.C

```diff
@@ -33,6 +33,8 @@ validParams<ElasticRecoilCrossSectionUserObject>()
       "elastic_xs", "Function representing the elastic cross section");
   params.addParam<unsigned int>(
       "legendre_order", 5, "Order of Legendre polynomials; Default to P5, where n = 0, ..., 5");
+  params.addParam<std::string>(
+      "output_file_name", "erxs_output.csv", "Name of the output file (.csv)");
   return params;
```

Comment:
I'd like you to add 
`params.addClassDescription("")` where the purpose and output format of the class is explained.